### DISCUSSION TITLE
Disclosure widget in PR doesn’t work before ”Read more”

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#4955] [Mobile]
Viewing <https://github.com/jenkinsci/warnings-ng-plugin/pull/998> in the GitHub 1.25.0 app on iOS 14.7.1:

![Pull request view with a dysfunctional "▶ Release notes" widget](https://user-images.githubusercontent.com/46201428/128601563-f662a22f-7d4a-48a3-af58-a8c569bb3f10.jpeg)

If I touch <kbd><samp>▶ Release notes</samp></kbd>, then I expect it to expand the release notes, but nothing happens.

However, if I scroll the page a bit, then I see a <kbd><samp>↓ Read more</samp></kbd> button:

![Pull request view scrolled to reveal the "↓  Read more" button](https://user-images.githubusercontent.com/46201428/128601673-ff4106f7-e86d-4feb-bd2d-9e435371d7ab.png)

If I touch <kbd><samp>↓ Read more</samp></kbd>, then <kbd><samp>▶ Release notes</samp></kbd> starts working.

So the UI eventually works but the behaviour is surprising because the <kbd><samp>▶ Release notes</samp></kbd> widget does not initially work even though it does not look disabled and the <kbd><samp>↓ Read more</samp></kbd> button is mostly off screen.

**Comment by eliperkins** [comment]
Argh, I hear ya @KalleOlaviNiemitalo, that is a frustrating bug. Thanks for this bug report! 🐛 

I've shared your notes about this bug with the engineering team. I'll keep you posted for when this bug gets fixed!
